lib/hammer_cli_foreman/hosts/common_update_options.rb

```diff
@@ -109,7 +109,6 @@ def domain_id(name)
       end
 
       def parameter_attributes
-        return [] unless option_parameters
```

Comment:
I think that we still want to keep this to enable intentional wipe of parameters.